cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/openshift/CreateBuildConfigOperation.java

```diff
@@ -0,0 +1,66 @@
+package io.strimzi.controller.cluster.operations.openshift;
+
+import io.fabric8.openshift.api.model.BuildConfig;
+import io.strimzi.controller.cluster.K8SUtils;
+import io.strimzi.controller.cluster.OpenShiftUtils;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Creates BuildConfig resource
+ */
+public class CreateBuildConfigOperation extends OpenShiftOperation {
+    private static final Logger log = LoggerFactory.getLogger(CreateBuildConfigOperation.class.getName());
+    private final BuildConfig build;
+
+    /**
+     * Constructor
+     *
+     * @param build BuildConfig which should be created
+     */
+    public CreateBuildConfigOperation(BuildConfig build) {
+        this.build = build;
+    }
+
+    /**
+     * @param vertx   Vert.x instance
+     * @param os      OpenShiftUtils instance
+     * @param handler Result handler
+     */
+    @Override
+    public void execute(Vertx vertx, OpenShiftUtils os, Handler<AsyncResult<Void>> handler) {
```

Comment:
From my perspective having the summary method in the base OpenShiftOperation class is enough. We don't need to have it every time we override that method.

Reply:
Hey ... either you want JavaDoc comments or not. :-) We should IMHO not crate complicated rules where not to put them and simply put them everywhere..

Reply:
But does it make sense having the summary when overriding a method ? Just my opinion :-(

Reply:
Well, in this case the execute actually does something different in each case, because the method is very generic (_execute_). Of course it should be clear what it does from the class name. But I think it might make sense.